treeherder/webapp/api/performance_data.py

```diff
@@ -1015,15 +1014,20 @@ def list(self, request):
                     "platform": platform,
                     "base_app": base_sig.get("application", ""),
                     "new_app": new_sig.get("application", ""),
-                    "suite": base_sig.get("suite", ""),  # same suite for base_result and new_result
-                    "test": base_sig.get("test", ""),  # same test for base_result and new_result
+                    "suite": base_sig.get("suite", "")
+                    if base_sig
+                    else new_sig.get("suite", ""),  # same suite for base_result and new_result
+                    "test": base_sig.get("test", "")
+                    if base_sig
+                    else new_sig.get("test", ""),  # same test for base_result and new_result
                     "is_complete": is_complete,
                     "framework_id": framework,
-                    "is_empty": is_empty,
```

Comment:
There's no need for this field, it's always False and is not actually used in any of the UI logic.